src/types/registration.types.ts

```diff
@@ -83,6 +83,10 @@ interface AdditionalIdentifier {
   value: string;
 }
 
+interface PublicationNote {
+  publicationNoteMessage: string;
```

Comment:
Denne typen ser ikke ut til å stemme?

```suggestion
  type: 'UnpublishingNote';
  note: string;
```

![bilde](https://github.com/BIBSYSDEV/NVA-Frontend/assets/6313468/74e672e6-8c0a-4b57-a518-e4e0144ccd07)

Reply:
Fikset typen, men ser at vi har flere typer publicationNote og en av de har publicationNoteMessage. Skal ta det opp med Torbjørn så vi får kun en note på disse.